.github/workflows/ci.yml

```diff
@@ -13,37 +13,40 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        cabal: ["3.2"]
+        cabal: ["3.4"]
         ghc:
-          - "8.10.1"
-          - "8.10.2"
+          - "9.0.0.20201227"
```

Comment:
Since we have a separate branch for every release, I don’t think there’s much point to specifying the ghc version in the matrix – could be moved directly into the `Setup Haskell` step